java/client/build.gradle

```diff
@@ -10,7 +10,7 @@ repositories {
 }
 
 dependencies {
-    implementation group: 'com.google.protobuf', name: 'protobuf-java', version: '4.26.1'
+    implementation group: 'com.google.protobuf', name: 'protobuf-java', version: '4.27.1'
```

Comment:
Should we align this with protoc binary we require? line 41

Reply:
Not necessary. This runtime works with protobuf 26.1.